Cargo.toml

```diff
@@ -22,12 +22,15 @@ num-complex = ">= 0.2, < 0.5"
 num-integer = "0.1"
 num-traits = "0.2"
 ndarray = ">= 0.13, < 0.16"
-pyo3 = { version = "0.21.0", default-features = false, features = ["macros"] }
+pyo3 = { version = "0.22.0", default-features = false, features = ["macros", "py-clone"] }
```

Comment:
py-clone should not be enabled unconditionally (perhaps behind a forwarding feature as with gil-refs) since it allows potentially panicking clones in any codebase that pulls in numpy as a dependency